vendor/vendor.json

```diff
@@ -555,6 +555,12 @@
 			"revisionTime": "2016-12-13T22:48:10Z"
 		},
 		{
+			"checksumSHA1": "+IGrULOjgHRwCPAk45iKxH3jznQ=",
+			"path": "github.com/smartystreets/assertions/should",
```

Comment:
Please use the `ShouldXXX` functions in `assertions` package, like we do everywhere else.